models/netsuite2/src_netsuite2.yml

```diff
@@ -2,246 +2,294 @@ version: 2
 
 sources:
   - name: netsuite2
-    schema: "{{ var('netsuite_schema', 'netsuite') }}" # you will need to customize if your schema name is not netsuite
+    schema: "{{ var('netsuite_schema', 'netsuite') }}"
     database: "{{ var('netsuite_database', target.database) }}"
     loader: fivetran
     loaded_at_field: _fivetran_synced
     config:
-      enabled: "{{ var('data_model', 'netsuite') == 'netsuite2' }}"
+      enabled: "{{ var('netsuite_data_model', 'netsuite') == var('netsuite2_variable_name','netsuite2') }}"
     freshness:
       warn_after: { count: 24, period: hour }
       error_after: { count: 48, period: hour }
 
     tables:
-      - name: accountingbook
-        description: Table detailing all accounting books set up in Netsuite.
+      - name: account_type
+        identifier: "{{ var('netsuite2_account_type_identifier', 'accounttype') }}"
+        description: "{{ doc('account_types_table') }}"
         columns:
-          - name: identifier
-            description: The unique identifier of the accounting book.
           - name: _fivetran_deleted
-            description: Boolean created by Fivetran to indicate whether the record has been deleted.
+            description:  "{{ doc('_fivetran_deleted') }}"
+          - name: _fivetran_synced
+            description:  "{{ doc('_fivetran_synced') }}"
+          - name: id
+            description: "{{ doc('account_type_id') }}"
+          - name: balancesheet
+            description: Boolean indicating if the account type is a balance sheet account.
```

Comment:
in the source tables, boolean fields are represented as string 'T' or 'F's, should we make that clear in the source.yml, though we convert them to actual booleans in the staging layer?

Reply:
Good point. I can update those accordingly.